vue/test/components/select/select.test.js

```diff
@@ -0,0 +1,324 @@
+        const dropdownSecondItem = component.findAll(".dropdown-item").at(1);
+        const dropdownThirdItem = component.findAll(".dropdown-item").at(2);
+        assert.strictEqual(dropdownItemsLength, 3);
+        assert.strictEqual(dropdownFirstItem.text(), "A");
+        assert.strictEqual(dropdownSecondItem.text(), "B");
+        assert.strictEqual(dropdownThirdItem.text(), "C");
+    });
+
+    it("should update value according to prop", async () => {
+        const component = base.getComponent("Select", {
+            props: {
+                options: [
+                    { value: "option_1", label: "A" },
+                    { value: "option_2", label: "B" },
+                    { value: "option_3", label: "C" }
+                ],
+                value: "option_1"
+            }
+        });
+        const componentData = component.vm.$data;
+        const selectButton = component.get(".select-button");
+        assert.strictEqual(selectButton.text(), "A");
+        assert.strictEqual(componentData.valueData, "option_1");
+
+        await component.setProps({ value: "option_2" });
+        assert.strictEqual(selectButton.text(), "B");
+        assert.strictEqual(componentData.valueData, "option_2");
+
+        await component.setProps({ value: "option_3" });
+        assert.strictEqual(selectButton.text(), "C");
+        assert.strictEqual(componentData.valueData, "option_3");
+
+        await component.setProps({ value: "option_1" });
+        assert.strictEqual(selectButton.text(), "A");
+        assert.strictEqual(componentData.valueData, "option_1");
+
+        await component.setProps({ value: null });
+        assert.strictEqual(componentData.valueData, null);
+
+        await component.setProps({ value: "option_2" });
+        assert.strictEqual(selectButton.text(), "B");
+        assert.strictEqual(componentData.valueData, "option_2");
+    });
+
+    it("should update visibility according to prop", async () => {
+        const component = base.getComponent("Select", {
+            props: {
+                visible: true
+            }
+        });
+        const componentData = component.vm.$data;
+        assert.strictEqual(componentData.visibleData, true);
+
+        await component.setProps({ visible: false });
+        assert.strictEqual(componentData.visibleData, false);
+    });
+
+    it("should update visibility on select button click", async () => {
+        const component = base.getComponent("Select");
+        const componentData = component.vm.$data;
+
+        const selectButton = component.get(".select-button");
+        const dropdown = component.findComponent({ name: "dropdown" });
+        const dropdownData = dropdown.vm.$data;
+
+        // toggles visibility on select-button click
+        assert.strictEqual(componentData.visibleData, false);
+        assert.strictEqual(dropdownData.visibleData, false);
+        await selectButton.trigger("click");

```

Comment:
Space between actions and assertions, like you did in the rest of the places, so that the end result is something like:

```
action1
assertion about action1
assertion about action1
assertion about action1

action2
assertion about action2
```